config/README-EN.md

```diff
@@ -0,0 +1,116 @@
+<h1 align="center">Platform Configuration Guide</h1>
+
+## Configuration Environment `./config`
+The configuration environment consists of a `public` directory, which will overwrite the original during the build process, and a TypeScript file for specific customizations and color definitions.
+
+In the build process, a script copies the root `public` directory to `dist/public`. After that, it checks if the `SVG`, `JSON`, `PNG` files and favicons in `config/public` are in their respective places, according to the original pattern and correctly named. If everything is correct, the files are overwritten in the `dist` folder.
+
+## SVG logo
+Placing the `SVG` logo in `config/public/images` with the standard name `logo1_white.svg` will overwrite the original during the build process.
```

Comment:
Question: Should we find a better standard file name ?

Reply:
I believe so, I would suggest "defaultLogo" what do you think?

Reply:
Sounds good, I'd recommend to use the same text case, e.g: `default_logo.svg`.